Makefile

```diff
@@ -65,7 +65,6 @@ test:
 	go vet ./...
 
 prepare:
```

Comment:
Why no t remove `prepare` completely?

Reply:
See third bullet in PR comments at the top.

Reply:
Checking with DevOps, but I think the empty make prepare will need to stay since the CI template also builds delhi and california branches which require CI to call make prepare.

Reply:
@tsconn23 , @jduranf , DevOps confirmed that the empty make prepare needs to stay in order to support the CI template build of delhi and california branches.

Reply:
@tsconn23 , all references to "go" need to be "$(GO)" so GO111MODULE=on is used. 

These are the targets that need to be updated:
build: $(MICROSERVICES)
	go build ./...

test:
	go test -cover ./...
	go vet ./...